rmgpy/data/kinetics/family.py

```diff
@@ -1448,17 +1448,11 @@ def addReverseAttribute(self, rxn):
         For rxn (with species' objects) from families with ownReverse, this method adds a `reverse`
         attribute that contains the reverse reaction information (like degeneracy)
         """
-        from rmgpy.rmg.react import findDegeneracies, getMoleculeTuples
+        from rmgpy.rmg.react import findDegeneracies
```

Comment:
I don't know if there's an official stance. It is also possible that there are issues with circular imports if this is moved to the top, but I haven't tested it. I can look into it though.

Reply:
Moving this import to the top of the file basically breaks all of RMG due to circular imports. I will see if organization improvements can be made as part of the fix for RMG-website functionality, but this will have to stay here for now.